openstack/run_packstack_docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
that's a good start but why not to run it within the cluster?

Reply:
kind already runs on its own  docker container , 
since this container consumes a lot of  resources i wanted to separate 
( In case  gh-action runner out-of-memory will kick-in ) , we can try to move it later.

Reply:
I've already reduced the amount of requested CPUs significantly in order to allow other, more heavy, workloads to run there - if we're able to run both the cluster and this container on a github hosted-runner, we can tweak the cluster even more if necessary to be able to run the container within the cluster. but ack, let's try this later after we see an end to end test works well